src/main/java/org/rumbledb/compiler/TranslationVisitor.java

```diff
@@ -158,14 +159,15 @@ public Node visitModuleImport(JsoniqParser.ModuleImportContext ctx) {
 
     @Override
     public Node visitFunctionDecl(JsoniqParser.FunctionDeclContext ctx) {
-        String fnName = ctx.fn_name.getText();
-        Map<String, SequenceType> fnParams = new LinkedHashMap<>();
+        // TODO namespace
```

Comment:
TODO

Reply:
This is intentional. I marked the places I need to modify next when I add namespace declaration support. Thanks for the check.